src/content/docs/apm/agents/manage-apm-agents/agent-data/manage-errors-apm-collect-ignore-or-mark-expected.mdx

```diff
@@ -13,11 +13,15 @@ redirects:
   - /docs/agents/manage-apm-agents/agent-data/manage-errors-apm-collect-ignore-mark-expected
 ---
 
-[APM](/docs/apm) agents automatically report error data for [supported frameworks](/docs/agents/manage-apm-agents/installation/compatibility-requirements). To optimize error reporting and alerting, you can further manage errors in order to:
+APM agents automatically report error data for [supported frameworks](/docs/agents/manage-apm-agents/installation/compatibility-requirements). To optimize error reporting and alerting, you can further manage errors in order to:
 
-* [Catch errors](#error-collection) that we don't instrument by default.
-* [Ignore errors](#ignore) that you don't want reported at all.
-* Filter out noise from [expected errors](#expected) so you can focus on the errors that are affecting performance. (Java, Ruby, Node, Python, and .NET agents only)
+* Catch errors that we don't instrument by default.
```

Comment:
Why did you take out the additional links @akristen?

Reply:
They were pretty noisy at the start of the doc, and people can use the quick nav on the right to jump to different sections of the doc. I think including a bunch of links at the start of the doc can create a lot of link fatigue and distract from just tackling the content yourself.